src/Illuminate/Foundation/Testing/DatabaseTransactions.php

```diff
@@ -4,12 +4,42 @@
 
 trait DatabaseTransactions
 {
+    /**
+     * This property will store references to the connections currently affected by the transaction. They are stored
+     * as a key => value array of connection_name => ConnectionObject.
+     *
+     * @var array
+     */
+    protected $transactingConnections = [];
+
+    /**
+     * Get the names of the database connections that should be included in the transaction. The default value of empty
```

Comment:
What exactly is the cs violation here? Style checks all pass, and there's nothing in the contribution guide except PSR-2.